tests/user_feedback.cpp

```diff
@@ -142,5 +144,55 @@ TEST_CASE("user feedback")
 		using namespace toml::literals; // should compile without namespace ambiguity
 		auto table = "[table]\nkey=\"value\""_toml;
 	}
+
+	SECTION("Using std::distance, std::count_if, etc. with Iterators") // No issue
```

Comment:
I'd prefer if the tests in this file were derived exclusively from user bug reports and feature requests (i.e. where there's a preceding dialogue with a user that I can link back to), so `user_feedback.cpp` perhaps isn't the place for it.

I'd suggest moving the table checks to `manipulating_tables.cpp` and the array checks to `manipulating_arrays.cpp`. Or create an iterator-specific test TU... I'm happy with either.

Reply:
Yeah I thought about that. But my conclusion was I was just not reporting the bug before fixing it. :)